cocktails9/cocktails9/register.swift

```diff
@@ -0,0 +1,147 @@
+//
+//  register.swift
+//  cocktails9
+//
+//  Created by Stasa Zujkovic on 27.11.24..
+//
+
+import SwiftUI
+
+struct register: View {
```

Comment:
Best practice in Swift suggests to name classes, stucts etc. with capital letter, also SwiftUI practice is to add suffix "View" so this would be best to name "RegisterView"